proposals/restructure-KF-docs-proposal.md

```diff
@@ -0,0 +1,228 @@
+This is an initial proposal, to be iterated on, following the discussion in [#2293](https://github.com/kubeflow/website/issues/2293).
+
+### Kubeflow documentation structure
+
+- **Getting Started**
+  - Introduction (current About/Kubeflow)
+  - Overview (current Kubeflow Overview)
+  - Use Cases (current About/Use Cases)
+  - Installing Kubeflow
+  - Cloud Installation
+    - AWS
+    - Azure
+    - Google Cloud
+    - IBM Cloud
+  - Kubernetes installation
+    - Overview of Deployment on Existing Clusters
+    - Kubeflow Deployment with kfctl_k8s_istio
+    - Multi-user, auth-enabled Kubeflow with kfctl_existing_arrikto
+    - Multi-user, auth-enabled Kubeflow with kfctl_istio_dex
+  - Workstation Installation
+    - Kubeflow on Linux
+    - Kubeflow on macOS
+    - Kubeflow on Windows
+    - MiniKF
+    - Deploy using MiniKF on GCP
+    - Deploying with minikube on a single node
+    - Kubeflow on MicroK8s
+    
+- **Community**
+  - About (current About/Community)
+  - Contributing (current About/Contributing to Kubeflow + details on Working Groups)
+  - Documentation (merge current About/Docs + About/Style Guide for Kubeflow docs)
+  - Events (current About/Events Calendar)
+  - Google Summer of Code (current About/Google Summer of Code)
+    
+- **Components** (remove "of Kubeflow")
+  - Jupyter Notebooks
+    - Overview of Jupyter Notebooks in Kubeflow
+    - Set Up Your Notebooks
+    - Create a Custom Jupyter Image
+    - Submit Kubernetes Resources
+    - Build a Docker Image on GCP
+    - Troubleshooting Guide
+  - Central Dashboard
+    - Central Dashboard 
+    - Registration Flow
+  - Metadata
+  - Fairing 
+    - Overview of Kubeflow Fairing
+    - Install Kubeflow Fairing
+    - Configure Kubeflow Fairing
+    - Fairing on Azure
+    - Fairing on GCP
+    - Tutorials
+    - Reference
+  - Feature Store
+    - Introduction to Feast
+    - Getting started with Feast
+  - Frameworks for Training
+    - Chainer Training
+    - MPI Training
+    - MXNet Training
+    - PyTorch Training
+    - TensorFlow Training (TFJob)
+  - Hyperparameter Tuning
+    - Introduction to Katib
+    - Getting started with Katib
+    - Running an experiment
+    - Katib Configuration Overview
+    - Environment Variables for Katib Components
+  - Pipelines
+    - Pipelines
+  - Tools for Serving
+    - Overview
+    - KFServing
+    - Seldon Core Serving
+    - BentoML
+    - NVIDIA Triton Inference Server
+    - TensorFlow Serving
+    - TensorFlow Batch Prediction
+  - Multi-Tenancy in Kubeflow
+    - Introduction to Multi-user Isolation
+    - Design for Multi-user Isolation
+    - Getting Started with Multi-user Isolation
+  - Miscellaneous
+    - Nuclio functions
+    
+- **Kubeflow Pipelines**
```

Comment:
Should `Pipelines` live under `Components`?

Reply:
+1, KFP is not a special candidate but first citizen of `Components`.

Reply:
If component applications will be moving to their own site, I don't think that it makes sense to move the Pipelines docs twice. I would recommend leaving Notebooks and Pipelines where they are until they move to their own sites.

Reply:
I don't quite understand, what's the reason that Kubeflow components applications should be moving to their own site rather than stay with kubeflow.org? 

We should recommend kubeflow components stay and push distribution owner move out.

Reply:
There are a few reasons that are leading this this decision. 

- Working groups are now (or soon will be) releasing updates on their own schedule, instead of releasing  as one Kubeflow release. 
- There have also been discussions around moving towards distributions of Kubeflow rather than releases, where each distribution could control the versions of components that are included. 
- Kubeflow components can be useful on their own.

To make it possible for users to get the correct documentation for their Kubeflow components, working groups need to be able to version the docs for their own releases. This lets distributions link to the correct docs for their components and it provides a place for documentation for users who only want to use one component. 

In order for the working groups to version their own docs, they will need to move off of www.kubeflow.org.

Reply:
Hi all! Thank you for the comments! 

@joeliedtke I agree with your points on releasing, versioning and hosting. Given that it is a bit orthogonal to this proposal and given that this comment is outdated I will resolve it so we can move ahead with the proposal.

Please do comment on anything you disagree with in the latest commit :)